src/cssVarsPonyfill.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * If the browser does NOT support CSS Variables
+ * loads and applies css-vars-ponyfill.
+ */
```

Comment:
Which browsers do not support CSS Variables?

Reply:
From looking at CanIUse.com of the browsers we support it's only IE11, which should be resolved by the ponyfill included in this PR
https://caniuse.com/?search=css%20variables